### PR TITLE
add trust dialog info

### PR DESCRIPTION
**Pull request by heyitsvi**

### CODE REVIEW

File: guide/sharing-a-pear-app.md [hunk 1]
```diff
@@ -93,7 +93,24 @@ With another machine or friend that has `pear` installed execute the `pear run <
 pear run pear://nykmkrpwgadcd8m9x5khhh43j9izj123eguzqg3ygta7yn1s379o
 ```
 
-When `pear run` is executed on the peer machine the staged application should open on that peer. If the application is being opened for the first time on the peer it has no state at all, so the application may show a loader until it's ready to reveal.
+When `pear run` is executed on the peer machine there will be a security prompt to add the key to a list of trusted applications by typing 'TRUST'.
+
+The trust dialog is a security mechanism in Pear that appears when the user tries to run an application from an unknown or untrusted key for the first time. In case that the app is run in detached mode, for example, when clicking on a pear link in the browser, the trust dialog is a GUI (Graphical User Interface). 
+
```
Comment (subashsn) [inline guide/sharing-a-pear-app.md]: Suggest adding a screenshot as example

File: guide/sharing-a-pear-app.md [hunk 2]
```diff
@@ -93,7 +93,24 @@ With another machine or friend that has `pear` installed execute the `pear run <
...
+> During development with `pear dev`, applications are automatically trusted, as they are assumed to be safe for testing purposes. Trust dialog can be avoided by using the `--no-ask-trust` flag with `pear run`. However, this is not recommended unless application's safety is guaranteed.
```
Comment (subashsn) [inline guide/sharing-a-pear-app.md]: This can be below the CLI flow below

File: guide/sharing-a-pear-app.md [hunk 3]
```diff
@@ -93,7 +93,24 @@ With another machine or friend that has `pear` installed execute the `pear run <
...
+
+```
```
Comment (subashsn) [inline guide/sharing-a-pear-app.md:100]: You can add a line saying, when starting an application in terminal you get a message, etc..

File: guide/sharing-a-pear-app.md [hunk 4]
```diff
@@ -93,7 +93,24 @@ With another machine or friend that has `pear` installed execute the `pear run <
...
+✖ Key pear://nykmkrpwgadcd8m9x5khhh43j9izj123eguzqg3ygta7yn1s379o is not known
+
+Be sure that software is trusted before running it
+
+Type "TRUST" to allow execution or anything else to exit
+
+Trust application? 
+```
+
+If the application is being opened for the first time on the peer it has no state at all, so the application may show a loader until it's ready to reveal.
```
Comment (subashsn) [inline guide/sharing-a-pear-app.md]: grammar issue

**Review by subashsn — CHANGES_REQUESTED** [review]

File: guide/sharing-a-pear-app.md (hunk 4 above)
Comment (heyitsvi) [reply guide/sharing-a-pear-app.md]: changes as requested, pushed

File: assets/image.png:Zone.Identifier
Comment (subashsn) [inline assets/image.png:Zone.Identifier:1]: ?

**Review by subashsn — APPROVED** [review]

**Review by rafapaezbas — APPROVED** [review]

**Pull request merged**